.github/workflows/pkg.pr.new.yaml

```diff
@@ -0,0 +1,50 @@
+name: Publish Approved PR
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  pull_request_review:
+    types: [submitted]
```

Comment:
@sadeghbarati  I wonder do we need to build and publish on push, and pull request.

Seems like 

```on:
  pull_request_review:
    types: [submitted]
```

will do just fine 😁

Reply:
Lol, make sense 🤣